analyzer/src/funTest/assets/projects/synthetic/maven-expected-output-app.yml

```diff
@@ -12,8 +12,8 @@ project:
     mapped:
       Apache License, Version 2.0: "Apache-2.0"
   vcs:
-    type: ""
-    url: ""
+    type: "Git"
```

Comment:
FYI, this test also passes when cherry-picked to `main` without any of the other changes in this PR... so I'm not sure what it proves.

Reply:
Yes, because this test only checks if the replacement works with `connection`, which was before not tested in a `funTest`.
IMO, using the connection is the preferred way by Maven, and the test did help me to debug the issue.

To test the same functionality with the `url` would require me to create an additional test project, which I'd like to avoid, as the functionality is also covered by unit tests.